docs/user/dashboard/dashboard.asciidoc

```diff
@@ -228,6 +228,13 @@ To view the data field summary information, navigate to the field, then click *i
 [role="screenshot"]
 image::images/lens_data_info.png[Data summary window]
 
+*Lens* inspects the top 10 values to build the value distribution: every other value, together with documents with no data for the inspected field, are shown with the "Other" entry in the distribution chart.
+
+[role="screenshot"]
+image::images/lens_data_info_other.png[Data summary window with Other]
+
+NOTE: because the distribution is based on single values, the sum of all the entries and "Other" may be above 100%, because entries in array values are counted as distinct values.
```

Comment:
I did a quick test and while Tim raises a good point in the issue you opened, this is not the case right now:
<img width="308" alt="Screenshot 2020-12-14 at 15 25 17" src="https://user-images.githubusercontent.com/1508364/102092676-9aaf7e00-3e20-11eb-8186-3471626216b1.png">

For this data:
```
POST testindex/_doc
{
  "val": ["A", "B", "C"]
}

POST testindex/_doc
{
  "val": ["D", "E", "F"]
}

POST testindex/_doc
{
  "val": ["G", "H", "I"]
}
```

Right now it seems like this only caused by rounding

Reply:
Catching up on this discussion: @flash1293 is right that it's only a rounding issue, not an issue with the underlying calculation. The size of "other" is based on the `value_count` of the field minus the sum of all terms we get back, which is correct for array fields and regular fields.